acceptance-test/src/test/java/com/poetry/domain/IRequestVerse.java

```diff
@@ -0,0 +1,5 @@
+package com.poetry.domain;
+
+public interface IRequestVerse {
```

Comment:
I know this old school way of putting I to denote interfaces. I think we can remove I. We will keep it simple just `RequestVerse` as IDEs shows what kind it is class | interface | enum. Olden days when people used notepads then maybe they wanted to prefix with I.